### ISSUE TITLE
Basic mining income calculation issues

### ISSUE THREAD

**Issue by jjchangg** [poseidon-network/qlauncher-linux#23]
I found one of my nodes that  "Basic mining income" is incorrect on 7/6 (Pic 1),  check the reason is IPFS service stopping(Pic 2). Actually, the service works as well(Pic3). And after that ( around 30 mins) everything is back to normal(Pic 4).

The question is whether the IPFS calculation also looks at 22 hours, or only at the point of settlement?
![Pic1](https://user-images.githubusercontent.com/65033954/124731071-7a721880-df44-11eb-8bf2-b9ff8b8cfe4d.jpg)
![Pic2](https://user-images.githubusercontent.com/65033954/124731077-7c3bdc00-df44-11eb-8b51-3e859d51003e.jpg)
![Pic3](https://user-images.githubusercontent.com/65033954/124731079-7cd47280-df44-11eb-9582-d8555d9e7d24.jpg)
![Pic4](https://user-images.githubusercontent.com/65033954/124731083-7d6d0900-df44-11eb-8d5e-e1db6dc5d440.jpg)

In addition, there is a phenomenon, the storage space of this node was floating in the two days. In the previous statement, suddenly became 0 GB and then look again in the morning, it slowly came back. The number is always up and down in yesterday.

**Comment by jjchangg** [comment]
Happen again at my another node on 7/9.

**Comment by vickygao77** [comment]
Have you tried all this to solve it?
https://www.notion.so/poseidonnetwork/What-can-I-do-when-the-IPFS-Service-is-not-running-b51650044b20404daccbd5ddedb44208

**Comment by jjchangg** [comment]
No. I restarted again & my 77GB usage was reset at next day(mid night).

vickygao77 ***@***.***> 於 2021年7月13日 週二 下午8:02寫道：
